testdata/src/main/java/com/hedvig/app/testdata/dashboard/builders/InsuranceDataBuilder.kt

```diff
@@ -125,6 +125,7 @@ class InsuranceDataBuilder(
                 statusPills = emptyList(),
                 detailPills = emptyList(),
                 gradientOption = TypeOfContractGradientOption.GRADIENT_ONE,
+                supportsAddressChange = true,
```

Comment:
Not quite sure about the mocked data. It feels like it being defaulted to true is fine for now, but if not, we'd need to go into each instantiation of this object and put true/false manually since we don't have giraffe to do it for us.
Is this good enough for now? What are your thoughts?

Reply:
You could allow it to be passed into the `InsuranceDataBuilder`, and default it there. I think that would be good enough 🙂

Reply:
Agreed!